index.html

```diff
@@ -0,0 +1,464 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+       <div class="header">
+          <h1 id="logo">Aziz Mejri</h1>
+          <div>
+            <button type="button" class="menu-bars">
+              <i class="fas fa-bars"></i>
+            </button>
+          </div>
+          
+          <nav id="desktop-nav">
+            <ul>
+              <li><a class="nav-link" href="#projects">Portoflio</a></li>
+              <li><a class="nav-link" href="#about-me">About</a></li>
+              <li>
+                <a class="nav-link" href="#contact">Contact <i class="far fa-envelope"></i></a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+        
+        <nav id="mobile" class="menu">
+          <div></div>
+          <ul class="list-menu">
+            <li><a href="#" rel="noopener noreferrer">Hello</a></li>
+            <li><a href="#" rel="noopener noreferrer">Portfolio</a></li>
+            <li><a href="#" rel="noopener noreferrer">About</a></li>
+            <li><a href="#" rel="noopener noreferrer">Contact</a></li>
+          </ul>
```

Comment:
While you made a good effort to implement the mobile menu, I noticed that the mobile menu does not disappear when the screen is toggled back to desktop size. I think you have to fix this to improve the User Experience (UI/UX).
The image below :point_down:  will help elucidate the point I am making I believe.

![image](https://user-images.githubusercontent.com/45668340/143767081-d870b8de-3dda-4978-a08a-af334189acfc.png)

Reply:
It is a good practice for a frontend developer to follow the Figma design given. Though your design is not bad, I think it is better to follow the given mobile menu design.
Check the image of the design below :point_down: 
![image](https://user-images.githubusercontent.com/45668340/143767329-01a0dc7d-29fc-4d36-9e29-b13291d2b689.png)

Please, find the link [here](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).

Reply:
![image](https://user-images.githubusercontent.com/45668340/143767673-0f2cd592-0383-419a-8af3-9b03a4dfcdca.png)

@Kalunge, I think you need to make a final touch on the position of the close button. It is too close to the edge of the screen. You may consider bringing it a bit down and to the left. This will enhance the User Experience.

Reply:
On it. Thanks